cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -314,8 +288,7 @@ private BuildOptions constructBuildOptions() {
             buildOptionsBuilder.targetDir(targetDir.toString());
         }
 
-        return buildOptionsBuilder.setConfigSchemaGen(configSchemaGen)
```

Comment:
Looks like the wrong changes have been picked when rebasing with master. Please check and add this back.

Reply:
Isnt this repeated twice? I thought it was a mistake and removed it since after `setSticky` the configSchemaGen is called as well.

Reply:
Ok I added it back in a52b38b